src/strategy/key_value.cpp

```diff
@@ -0,0 +1,11 @@
+#include "key_value.h"
```

Comment:
If I move this to be inlined into the header file, i don't get the error. 

otherwise it tells me:
```
ERROR: ld.so: object './bazel-bin/src/main/libold.so' cannot be loaded as audit interface: undefined symbol: _ZN16KeyValueStrategyC1Ev; ignored.
```

@twrs